USAGE.rst

```diff
@@ -0,0 +1,137 @@
+kafkacat
+========
+
+kafkacat is a command line tool for working with Kafka. Described as "netcat for Kafka", it is a swiss-army knife of tools for inspecting and creating data in Kafka.
+
+If you already know ``kafka-console-producer`` and ``kafka-console-consumer`` then it's like those, on steroids.
+
+kafkacat is an open-source tool, available at https://github.com/edenhill/kafkacat. It is not included in Confluent Platform.
+
+``kafkacat`` the Consumer
```

Comment:
i'd suggest not using command syntax in the title. the output is a bit strange:
![image](https://user-images.githubusercontent.com/11722533/38324184-62ef80da-37f4-11e8-9fbf-051468c87c95.png)